doozerlib/dblib.py

```diff
@@ -1,17 +1,144 @@
 from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, print_function, unicode_literals
+from future.utils import as_native_str
+from abc import ABC, abstractmethod
+from typing import Any, Optional
 import time
 import os
 import sys
 import threading
 import hashlib
-import boto3
 import traceback
+import mysql.connector as mysql_connector
+from doozerlib import constants
+import functools
 from .model import Missing
+from dateutil.parser import parse
+
+
+class TypeIdentifiedHandler(ABC):
+
+    """
+    This interface declares methods for building chain of handlers for identifying the correct column type.
+    The interface also declares methods to handle the incoming request.
+    Reference: https://refactoring.guru/design-patterns/chain-of-responsibility/python/example
+    """
+
+    @abstractmethod
+    def set_next_handler(self, handler):
+        pass
+
+    @abstractmethod
+    def identify(self, request):
+        pass
+
+
+class AbstractTypeIdentifierHandler(TypeIdentifiedHandler):
+
+    """
+    Default chaining handler class with default methods.
+    """
+
+    _next_handler: TypeIdentifiedHandler = None
+
+    def set_next_handler(self, handler):
+        self._next_handler = handler
+        return handler
+
+    @abstractmethod
+    def identify(self, request: Any):
+
+        if self._next_handler:
+            return self._next_handler.identify(request)
+
+
+class IntegerIdentifier(AbstractTypeIdentifierHandler):
+    def identify(self, request: Any):
+        try:
+            int(request)
+
+            if len(request) <= 6:
+                return "INT", int(request)
+            return "BIGINT", int(request)
```

Comment:
Always use BIGINT. The first value I send you has no bearing on the size of the second value I send you.